text/src/autogluon/text/automm/predictor.py

```diff
@@ -1063,6 +1075,12 @@ def load(
             df_preprocessor = pickle.load(fp)
         with open(os.path.join(path, "data_processors.pkl"), "rb") as fp:
             data_processors = pickle.load(fp)
+        if TEXT in data_processors:
+            # Load text processors separately due to tokenizers.
+            data_processors[TEXT] = load_text_processors(
+                relative_paths=data_processors[TEXT],
+                path=path,
+            )
```

Comment:
We need to also support loading a checkpoint trained via `0.4.0`. We can detect the version and give special treatment on the 0.4.0-version models.